orchestra/utils/task_lifecycle.py

```diff
@@ -548,11 +549,15 @@ def tasks_assigned_to_worker(worker):
             next_todo_dict = {}
             should_be_active = False
             if state in ('returned', 'in_progress'):
+                # TODO(aditya): Temporarily we are filtering todos
+                # with section values. Remove this comment once we
+                # figure out a long term logic.
                 next_todo = (
                     task_assignment.task.todos
                     .filter(
-                        completed=False,
-                        template=None
+                        status=Todo.Status.PENDING.value,
```

Comment:
curiously, is there anywhere else in the code where we reference `completed` instead of `status`?

Reply:
I did look into it.I ran a search for `completed` and `skipped_datetime` and made sure we did not miss anything.

Reply:
Amazing thank you!